php/php.ini

```diff
@@ -38,4 +39,6 @@ opcache.use_cwd=1
 
 ; nescessary for file uploads
 upload_max_filesize = 32M
-post_max_size = 32M
\ No newline at end of file
+post_max_size = 32M
+
+extension=php_gd.dll
```

Comment:
Shouldn't it be `extension=gd`?

Reply:
In older versions including PHP 8.0, this still works. I made it into how it is in the documentation for future-proofing.